### PR TITLE
removed related rule that has been archived

### PR DESCRIPTION
**Pull request by tiagov8**

### CODE REVIEW

File: rules/build-cross-platform-apps/rule.md [hunk 1]
```diff
@@ -6,7 +6,6 @@ authors:
   - title: Matt Goldman
     url: https://www.ssw.com.au/people/matt-goldman
 related:
-  - use-net-maui
   - why-react-is-great
```
Comment (matt-goldman) [inline rules/build-cross-platform-apps/rule.md:9]: Why is this rule related? I don't think we should refer to React in this rule.

**Review by matt-goldman — COMMENTED** [review]
This change itself is fine but not sure why the React rule is related

File: rules/build-cross-platform-apps/rule.md (hunk 1 above)
Comment (tiagov8) [reply rules/build-cross-platform-apps/rule.md:9]: @matt-goldman It's mentioned on line 41... but please remove the related if you think it's not helpful

**Review by adamcogan — APPROVED** [review]
